### PR TITLE
Add support for I2C & connecting FFB Servos through PCA9685 16ch PWM Board

### PR DESCRIPTION
**Pull request by Roastlawyer**

### CODE REVIEW

File: open-gloves/Config.h [hunk 1]
```diff
@@ -112,6 +112,27 @@
...
+#define USE_PCA9685_16CH_FOR_HAPTICS    (false && ENABLE_PCA9685_16CH_PWM_BOARD)     // Requires ENABLE_PCA9685_16CH_PWM_BOARD to be true in order to work    (PLACEHOLDER Setting ATM)
+
+//If you have the servos connected directly to the microcontroller, configure using the PIN_*Finger*_FFB setting in the appropriate section for your microcontroller.
+#define SERVO_CH_PINKY_FFB   15  // Board CHs used for force feedback servos when connected through PCA9685 Servo Board
+#define SERVO_CH_RING_FFB    14 //^
+#define SERVO_CH_MIDDLE_FFB  13 //^
+#define SERVO_CH_INDEX_FFB   12 //^
+#define SERVO_CH_THUMB_FFB   11 //^
+
+#define PCA_9685_MIN_SERVOPULSE 100 //(total range should probably be 100ish-600ish)  start with values of 125 ish for Min and 500 for max, slowly decrease and increase them respectivly ↓
```
Comment (Roastlawyer) [inline open-gloves/Config.h:131]: A "PCA_9685_MIN_SERVOPULSE" value of 125 ticks and "PCA_9685_MAX_SERVOPULSE" of 500 ticks (out of 4096 per cycle) sends out a MinPulse that is about 600 microseconds and a MaxPulse of about 2400 microseconds (when using 50hz/20 millisecond PWM cycles)

**Review by JohnRThomas — CHANGES_REQUESTED** [review]
Please squash this into a single commit instead of 5.

File: open-gloves/ForceFeedback.hpp [hunk 2]
```diff
@@ -47,12 +51,28 @@ class ServoForceFeedback : public ForceFeedback {
 
   void setupOutput() override {
     // Initialize the servo and move it to the unrestricted base limit.
-    servo.attach(servo_pin);
-    servo.WRITE_FUNCTION(SERVO_MIN);
+    #if USE_PCA9685_16CH_FOR_FFB
```
Comment (JohnRThomas) [inline open-gloves/ForceFeedback.hpp:54]: Can you abstract this to a class that matches the interface of Servo and turn on and off functionality via a different include similar to how Servo vs ESP32Servo works in the current code.

Comment (Roastlawyer) [reply open-gloves/ForceFeedback.hpp:54]: I do not know how to do that, especially to allow the servo.attach to work in the way it does now. I might be able to condence down to servo.WRITE_FUNCTION(servo_Pin, SERVO_MIN) but I don't know how to replicate the caching "which pin to write to" behavior of servo.attach

**Pull request closed without merging**